src/frameworks/index.ts

```diff
@@ -538,6 +539,12 @@ ${
 
       // TODO move to templates
 
+      if (frameworksBackend && framework === "next") {
+        if (await isUsingImageOptimization(getProjectPath(), ".next")) {
```

Comment:
don't we need to do this only for Next 15?

Reply:
The memory issue is happening on Next 14 and 13 as well 🫠